### DISCUSSION TITLE
Possible Nintendo switch release? (Suggestion)

### DISCUSSION THREAD

**Discussion by SkylerDaKat** [ppy/osu#24244] [Ideas] [answered]
(putting it short cuz cats unplugged my device)
I want an official eShop release of osu since the only way to play osu and lazer is through homebrew, and I and alot people don't wanna do that,
I know there's issues such as the switch being a weak console and well, licensing issues + extra development time so I'm not expecting a yes answer
Anyways, have a good day :3

**Comment by peppy** [comment, marked as answer]
Maybe further down the line. We already have enough platforms to worry about.

**Reply by SkylerDaKat** [reply]
Yeah, very true.
Would not want lazer to become development hell xD
Anyways thx for the answer :3 
Hopefully down the line it can be possible but no stress,
Stay awesome peppy <3333
